source/_integrations/pi_hole.markdown

```diff
@@ -40,3 +40,25 @@ Disables configured Pi-hole(s) for the specified amount of time.
 | ---------------------- | -------- | -------- | ----------- |
 | `entity_id` | `False` | string | Target switch entity. Use `all` to target all Pi-hole services |
 | `duration` | `True` | timedelta | Time for which Pi-hole should be disabled |
+
+You can add the following to your `configuration.yaml` file if want to create a template switch which will disable Pi-Hole temporarily as opposed to turning off the main `switch` component provided by the integration. You can turn the switch on before the timer expires for an immediate effect or just leave it to turn itself on after timer expires.
+
+```yaml
+# Example configuration.yaml entry
+switch:
+  - platform: template
+    switches:
+      pihole_temp_disable:
+        friendly_name: "Disable Pi-Hole for 30 min"
```

Comment:
This feels weird to do, why not use an automation with a input_boolean for that?
Feel weird to have a template for this.

Additionally, please make sure the CI passes, thanks 👍

Reply:
Frankly, I have no idea what input boolean is :) I just wanted to save beginners like me from spending 2 hours trying to call this service that has no example given. New users don't know how to format `timedelta` type and searching for "Home Assistant timedelta" gives mostly references to Python timedelta() function...Also, new users don't know how to format a service call. Forgive me, but I thought quick and dirty example is better than no example. And for me this achieves exactly what I wanted in terms of functionality. I'd be happy to see alternatives, but I don't know how to make them. 
Anyway, it seems that checks are now passing.

Reply:
> Frankly, I have no idea what input boolean is :) I just wanted to save beginners like me from spending 2 hours trying to call this service

Let make an example service call in that case, and not a template switch (which IMHO isn't a great example or a method I would use/recommend personally).